SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRepNotify/SpatialTestRepNotify.cpp

```diff
@@ -162,66 +210,51 @@ void ASpatialTestRepNotify::PrepareTest()
 			{
-				if (OldTestArray.Num() != 2)
-				{
-					FinishTest(EFunctionalTestResult::Failed, TEXT("OnRepTestArray should have been called with 2 entries in the old Array "
-																   "on Spatial or in Native on 4.26 and above"));
-					return;
-				}
+				RequireEqual_Int(TestActor->OldTestArray.Num(), 2,
+								 TEXT("OnRepTestArray should been called with 2 entries in the old Array "
+									  "on Spatial or in Native on 4.26 and above"));
 			}
 			else
 			{
-				if (OldTestArray.Num() != 3)
-				{
-					FinishTest(EFunctionalTestResult::Failed,
-							   TEXT("OnRepTestArray should have been called with 3 entries in the old Array on Native in 4.25 and below"));
-					return;
-				}
-
-				if (OldTestArray[2] != 0)
+				if (RequireEqual_Int(TestActor->OldTestArray.Num(), 3,
+									 TEXT("OnRepTestArray should be called with 3 entries in the old Array on Native in 4.25 and below")))
 				{
-					FinishTest(EFunctionalTestResult::Failed, TEXT("OnRepTestArray should have been called with 0 as its third entry in "
-																   "the old Array on Native in 4.25 and below"));
-					return;
+					RequireEqual_Int(TestActor->OldTestArray[2], 0,
+									 TEXT("OnRepTestArray should be called with 0 as its third entry in "
+										  "the old Array on Native in 4.25 and below"));
 				}
 			}
 
-			if (OldTestArray[0] != 1)
+			if (TestActor->OldTestArray.Num() >= 2)
 			{
-				FinishTest(EFunctionalTestResult::Failed,
-						   TEXT("OnRepTestArray should have been called with 1 as its first entry in the old Array"));
-				return;
-			}
-
-			if (OldTestArray[1] != 2)
-			{
-				FinishTest(EFunctionalTestResult::Failed,
-						   TEXT("OnRepTestArray should have been called with 2 as its second entry in the old Array"));
-				return;
+				RequireEqual_Int(TestActor->OldTestArray[0], 1,
+								 TEXT("OnRepTestArray should be called with 1 as its first entry in the old Array"));
+				RequireEqual_Int(TestActor->OldTestArray[1], 2,
+								 TEXT("OnRepTestArray should be called with 2 as its second entry in the old Array"));
 			}
 
 			FinishStep();
 		},
-		nullptr, 5.0f);
+		5.0f);
 
 	AddStep(TEXT("SpatialTestRepNotifyServerTestRemovalFromArray"), FWorkerDefinition::Server(1), nullptr, [this]() {
-		TestArray.Pop(true);
+		TestActor->TestArray.Pop(true);
 
 		FinishStep();
 	});
 
 	AddStep(
-		TEXT("SpatialTestRepNotifyClientsCheckArrayRemoval"), FWorkerDefinition::AllClients,
-		[this]() -> bool {
-			// First make sure that we have correctly received the replicated variables, before checking RepNotify behaviour.
-			return TestArray.Num() == 2;
-		},
-		[this]() {
-	// At this point, we have received the update for the TestArray, so it makes sense to check RepNotify beahviour.
+		TEXT("SpatialTestRepNotifyClientsCheckArrayRemoval"), FWorkerDefinition::AllClients, nullptr, nullptr,
+		[this](float DeltaTime) {
+			// First make sure that we have correctly received the replicated variables
```

Comment:
This is more indented than the rest?

Reply:
whoopsy, fixed

Reply:
nvm - the stuff that was indented less was the comment below this. I fixed but turns out it's the formatter, probably some funny interaction with the `#if ENGINE_MINOR_VERSION >= 26` below the comment